app/code/Magento/Backend/Block/Widget/Tabs.php

```diff
@@ -235,32 +235,88 @@ protected function _setActiveTab($tabId)
      */
     protected function _beforeToHtml()
     {
+        $this->_tabs = $this->reorderTabs();
+        
+        if ($this->_activeTab === null) {
+            /** @var TabInterface $tab */
+            foreach ($this->_tabs as $tab) {
+                $this->_activeTab = $tab->getId();
+                break;
+            }
+        }
+        
         if ($activeTab = $this->getRequest()->getParam('active_tab')) {
             $this->setActiveTab($activeTab);
         } elseif ($activeTabId = $this->_authSession->getActiveTabId()) {
             $this->_setActiveTab($activeTabId);
         }
-
-        $_new = [];
+        
+        $this->assign('tabs', $this->_tabs);
+        return parent::_beforeToHtml();
+    }
+    
+    
+    /**
+     * @return array
+     */
+    protected function reorderTabs()
```

Comment:
Due to [Magento backward-compatible guide](http://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/) we can't create new protected methods.